balena/models/tag.py

```diff
@@ -392,27 +396,30 @@ def set(self, release_id, tag_key, value):
             OK: if tag exists.
 
         Examples:
-            >>> balena.models.tag.release.set('465307', 'releaseTag1', 'Python SDK')
+            >>> balena.models.tag.release.set(465307, 'releaseTag1', 'Python SDK')
             {u'release': {u'__deferred': {u'uri': u'/balena/release(465307)'}, u'__id': 465307}, u'tag_key': u'releaseTag1', u'id': 135, u'value': u'Python SDK', u'__metadata': {u'type': u'', u'uri': u'/balena/release_tag(135)'}}
-            >>> balena.models.tag.release.set('465307', 'releaseTag1', 'Python SDK 1')
+            >>> balena.models.tag.release.set(465307, 'releaseTag1', 'Python SDK 1')
             OK
 
         """
 
+        release_id = self.release.get(commit_or_id)['id']
+
         return super(ReleaseTag, self).set(release_id, tag_key, value)
 
-    def remove(self, release_id, tag_key):
+    def remove(self, commit_or_id, tag_key):
         """
         Remove a release tag.
 
         Args:
-            release_id (str): release id.
+            commit_or_id: release commit (str) or id (int).
             tag_key (str): tag key.
 
         Examples:
-            >>> balena.models.tag.release.remove('135', 'releaseTag1')
+            >>> balena.models.tag.release.remove(135, 'releaseTag1')
             OK
 
         """
 
+        release_id = self.release.get(commit_or_id)['id']
```

Comment:
> Should we now mention that this also throws an AmbiguousReleaseError?

Also, did this throw 404's before this change when called w/ a random id?

It might even make sense to have an `is_id` check in here and only `self.release.get` if it's a

Reply:
@thgreasi, `release.get` does use `is_id` to check the parameter value first and throw AmbiguousRelease and ReleaseNotFound exception so we don't need to repeat that in the above tag methods

Reply:
Oh ok, didn't know that it's inferred automatically :+1: